Formula/onelogin.rb

```diff
@@ -0,0 +1,35 @@
+class Onelogin < Formula
+  desc "CLI for Using OneLogin"
+  homepage "https://github.com/onelogin/onelogin"
+  url "https://github.com/onelogin/onelogin/archive/refs/tags/v0.1.14.tar.gz"
+  sha256 "d25d41b51f4a1464675b88e7de8026331d2a1ac2611674a9064535ccd5001ba5"
+  license "Apache-2.0"
+  head "https://github.com/onelogin/onelogin.git"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  bottle :unneeded
```

Comment:
```suggestion
```
We actually do need a bottle here.

Reply:
Which bottles do I need then? I'm not clear on where to get that info.